.github/workflows/sanity_check.yml

```diff
@@ -29,10 +29,8 @@ jobs:
     - name: Install argparse
       run: pip install argparse
 
-    - name: Download clang-format-diff.py
-      uses: wei/wget@v1
-      with:
-        args: https://raw.githubusercontent.com/llvm/llvm-project/main/clang/tools/clang-format/clang-format-diff.py
+    - name: Install clang-format
+      run: sudo apt-get install clang-format
```

Comment:
The clang-format downloaded here can still be a different version and generate different result, depending on the instance provided by Github.
How about stable google style?
```
https://raw.githubusercontent.com/llvm-mirror/clang/google/stable/tools/clang-format/clang-format-diff.py
```

Reply:
The repo is marked archived: https://github.com/llvm-mirror/clang/tree/google/stable and it's version will be too old.
How about we ping it to a certain version like:
```
apt-get install clang-format-11
```

Reply:
Then I think it better just use the tool from a release branch instead of main. Updated.